test/clients/channel-access-token/Api/ChannelAccessTokenApiTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
I realized that since it was a test for a different API class, it should be in a separate test class :pray: